app/packs/entrypoints/decidim_email.scss

```diff
@@ -0,0 +1,5 @@
+@import "stylesheets/decidim/email.scss";
+
+// Override styles
+.cityhall-bar { background-color: #ececec !important ; }
```

Comment:
is the !important necessary?

Reply:
Good question.
I would take a look at how SASS deals with multiple declarations for the same target.

Reply:
most likely it is not necessary. It never is

Reply:
Agree, since we are controlling how the order of declarations we don't need the `!important`, the latter ones will take precedence.

I'll address this.